tfhe/examples/utilities/generates_test_keys.rs

```diff
@@ -80,13 +83,13 @@ fn client_server_keys() {
             ShortintKeySwitchingParameters,
         ); 2] = [
             (
-                PARAM_MESSAGE_1_CARRY_1_KS_PBS,
-                PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+                PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
+                PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
```

Comment:
this is not what was tested before

Reply:
Yeah for this one I didn't really know what to do. What do you think we should have here? all Gaussians + the TUniform 2_2?